core/src/mindustry/content/Fx.java

```diff
@@ -159,7 +159,7 @@
     commandSend = new Effect(28, e -> {
         color(Pal.command);
         stroke(e.fout() * 2f);
-        Lines.circle(e.x, e.y, 4f + e.finpow() * 120f);
+        Lines.circle(e.x, e.y, 4f + e.finpow() * e.rotation * 0.8f);
```

Comment:
It seems misleading to make the displayed radius smaller than the actual radius - why multiply?

Reply:
Well, the command radius was 150 before, but the effect would only go up to 4 + 120.